superset/db_engine_specs.py

```diff
@@ -686,6 +805,50 @@ def fetch_result_sets(cls, db, datasource_type, force=False):
         return BaseEngineSpec.fetch_result_sets(
             db, datasource_type, force=force)
 
+    @staticmethod
+    def upload_csv(form, table):
+        "uploads a cv file and creates a superset table in the right databse"
+        "returns true if it was siccessful. False otherwise"
+        def get_column_names(filepath):
+            import csv
```

Comment:
imports go at the top of the module, linter should trigger on this

Reply:
Done